REST Bindings/openapi.yaml

```diff
@@ -4333,6 +4644,802 @@ components:
+      in: query
+      style: pipeDelimited
+      explode: false
+      required: false
+      schema:
+        type: array
+        minItems: 1
+        items: 
+          type: string
+          format: URI
+
+    MATCH_inputEPCClass:
+      name: MATCH_inputEPCClass
+      description: "If this parameter is specified, the result will only include events that (a) have an `inputQuantityList` field (that is, `TransformationEvent` or extension event types that extend it); and where (b) one of the EPC classes listed in the `inputQuantityList` field (depending on event type) matches one of the EPC patterns or URIs specified in this parameter. The meaning of 'matches' is as specified in section 8.2.7.1.1"
+      in: query
+      style: pipeDelimited
+      explode: false
+      required: false
+      schema:
+        type: array
+        minItems: 1
+        items: 
+          type: string
+          format: URI
+
+    MATCH_outputEPCClass:
+      name: MATCH_outputEPCClass
+      description: "If this parameter is specified, the result will only include events that (a) have an `outputQuantityList` field (that is, `TransformationEvent` or extension event types that extend it); and where (b) one of the EPC classes listed in the `outputQuantityList` field (depending on event type) matches one of the EPC patterns or URIs specified in this parameter. The meaning of 'matches' is as specified in section 8.2.7.1.1"
+      in: query
+      style: pipeDelimited
+      explode: false
+      required: false
+      schema:
+        type: array
+        minItems: 1
+        items: 
+          type: string
+          format: URI
+
+    MATCH_anyEPCClass:
+      name: MATCH_anyEPCClass
+      description: "If this parameter is specified, the result will only include events that (a) have a `quantityList`, `childQuantityList`, `inputQuantityList`, or `outputQuantityList` field (that is, `ObjectEvent`, `AggregationEvent`, `TransactionEvent`, `TransformationEvent`, `AssociationEvent` or extension event types that extend one of those event types); and where (b) one of the EPC classes listed in any of those fields matches one of the EPC patterns or URIs specified in this parameter. The result will also include `QuantityEvent`s whose `epcClass` field matches one of the URIs specified in this parameter. The meaning of 'matches' is as specified in section 8.2.7.1.1."
+      in: query
+      style: pipeDelimited
+      explode: false
+      required: false
+      schema:
+        type: array
+        minItems: 1
+        items: 
+          type: string
+          format: URI
+
+    EQ_quantity:
+      name: EQ_quantity
+      description: "(DEPCRECATED in EPCIS 1.1, REPURPOSED in EPCIS 2.0) If this parameter is specified, the result will only include events that (a) have a `quantity` field as part of a `QuantityElement`; and where (b) the `quantity` field is equal to the specified parameter."
+      in: query
+      required: false
+      schema:
+        type: integer
```

Comment:
could quantity be a float?

Reply:
Good catch! @jmcanterafonseca-iota .  Alerting @CraigRe to another inconsistency in the draft spec.
The UML class diagrams show quantity as xsd:double
The ontology files show quantity as xsd:decimal
Section 7.3.3.1 (QuantityElement) shows quantity as 'Decimal' - though it's unclear whether that is actually xsd:decimal
Section 8.2.7.1 (SimpleEventQuery) shows EQ_quantity expecting an 'Int' , while EQ_quantity_uom is a List of String

So much for any consistency within the draft spec!